From 3d443d5b17f5c8b86cd42608beca8193c69969be Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 4 Mar 2020 00:26:04 +0100
Subject: [PATCH] extensionPrefs: Stop escaping extension name

The corresponding label no longer uses markup, so we can and should use
the unescaped name.

https://gitlab.gnome.org/GNOME/gnome-shell/issues/2305
---
 js/extensionPrefs/main.js | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/js/extensionPrefs/main.js b/js/extensionPrefs/main.js
index 1ae010b361..02e68c76d6 100644
--- a/js/extensionPrefs/main.js
+++ b/js/extensionPrefs/main.js
@@ -633,8 +633,7 @@ var ExtensionRow = GObject.registerClass({
         });
         this._actionGroup.add_action(action);
 
-        let name = GLib.markup_escape_text(this.name, -1);
-        this._nameLabel.label = name;
+        this._nameLabel.label = this.name;
 
         let desc = this._extension.metadata.description.split('\n')[0];
         this._descriptionLabel.label = desc;
-- 
2.24.1

